client/app/pages/queries/query.html

```diff
@@ -248,7 +248,7 @@ <h3>
             </div>
             <div class="row">
               <div class="col-lg-12">
-                <div ng-if="!query.visualizations.length">
+                <div ng-if="!query.visualizations.length" class="query__vis col-md-12 p-b-15">
```

Comment:
Did you need to apply anything similar on the dashboard page?

Reply:
This one fixes the unsaved query alignment. The filter headers are fixed with this change: https://github.com/getredash/redash/pull/2200/commits/4c055c663be6c3b679622fc9b5f3034d1630af97#diff-e8061505c2069e286392a8ad98acdb53

Reply:
👍

Reply:
I messed up some stuff here so I'm fixing them here: https://github.com/getredash/redash/pull/2201